app/src/main/java/ch/epfl/sweng/swenggolf/messaging/MessagesAdapter.java

```diff
@@ -0,0 +1,92 @@
+package ch.epfl.sweng.swenggolf.messaging;
+
+import android.net.Uri;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.squareup.picasso.Picasso;
+
+import java.util.ArrayList;
+
+import ch.epfl.sweng.swenggolf.Config;
+import ch.epfl.sweng.swenggolf.R;
+import ch.epfl.sweng.swenggolf.database.ValueListener;
+import ch.epfl.sweng.swenggolf.offer.answer.Answer;
+import ch.epfl.sweng.swenggolf.offer.answer.Answers;
+import ch.epfl.sweng.swenggolf.offer.answer.ListAnswerAdapter;
+import ch.epfl.sweng.swenggolf.profile.User;
+
+public class MessagesAdapter extends RecyclerView.Adapter<ListAnswerAdapter.AnswerViewHolder> {
```

Comment:
Why do you redefine all methods, can't you just get the already defined ones of ListAnswerAdapter ? And if not why don't you call the ones of the answers you stored instead of duplicating variables. For examples getUpdateListener might return answers.getUpdateListener().

Reply:
Okay, I mistook Answers and the adapter. Yeah it's fine this way !

Reply:
Okay, I mistook Answers and the adapter. It's fine this way.